src/org/folio/Constants.groovy

```diff
@@ -57,7 +57,7 @@ class Constants {
 
     //PostgreSQL
     static String PSQL_DUMP_HELM_CHART_NAME = "psql-dump"
-    static String PSQL_DUMP_HELM_INSTALL_CHART_VERSION = "1.0.2"
+    static String PSQL_DUMP_HELM_INSTALL_CHART_VERSION = "1.0.3-RANCHER-503-20221031151231"
```

Comment:
Is it correct version? Maybe 1.0.3 should be?

Reply:
Yep, sure)
I'll update it)

Reply:
Done)